min/lib/Minify.php

```diff
@@ -494,6 +494,13 @@ protected static function _combineMinify()
             $source = null;
             if ($i < $l) {
                 $source = self::$_controller->sources[$i];
+                
+                //Change the minifier to disable the compression
+                if(!empty(self::$_options['concatOnly'])){
+                    $source->minifier = '';
```

Comment:
This will break CSS URL rewriting sources. Also the sources must be altered much earlier in serve(). Otherwise these will generate the same cache ID as the normal sources.